app/javascript/components/greetings.jsx

```diff
@@ -1,22 +1,18 @@
 import React, {useEffect} from 'react'
-    import { useDispatch, useSelector } from "react-redux";
-    import { getAllGreets, fetchGreetingAsyncThank } from "../redux/greetReducer";
+    import {useDispatch, useSelector} from "react-redux";
+    import {fetchGreetingAsyncThank} from "../redux/greetReducer";
     const Greeting = () => {
-        const greetList = useSelector(getAllGreets);
-        console.log(greetList)
-        const dispatch = useDispatch();
-        useEffect(() => {
-            if (greetList.length === 0 )
+        const random = Math.floor(Math.random() * 5);
+            const message = useSelector((state) => state.greetReducer);
+            const dispatch = useDispatch();
+            useEffect(() => {
                 dispatch(fetchGreetingAsyncThank());
-        }, [dispatch]);
-
-        return <ul>
-            Top {greetList.length} greetings
-            {greetList.map((greet) => (
-                    <li>{greet.message}</li>
-                )
-            )}
-        </ul>;
+            }, []);
+        return (
+            <>
```

Comment:
- [ ] As requested by the previous reviewer please fix this problem 😉 

![image](https://user-images.githubusercontent.com/75126481/223858293-ce9a8b8a-a86b-4695-b73d-8b5d69569d5d.png)

Reply:
Hello @hiromataba,

This is because of the npm dependencies... I think You forgot to run the `npm install` and start the webpack in watch mode `yarn build --watch`. that's why you're seeing this error. It perfectly run's on my side

Reply:
@hiromataba,

Kindly, before you start the rails server, You have to run `npm install` and  then  `yarn build --watch`

Reply:
@degisew  did check yeah,
I think my npm dépendances did not run well,also it's better to always add all descriptions how to run your app 😃 

#Important note,

Please for the next reviewer kindly approve this project

Reply:
@hiromataba

Thanks a lot @hiromataba

Reply:
# Note for Next Reviewer

Dear Next Reviewer, I am asking for this Review because of miss understanding from the previous reviewer. My project was right but while the previous reviewer reviewing,  he forget to run `npm install` and yarn build --watch on his local machine before he start the rails server and he got an error.  that's why he asked for changes on my repo.